Global.gd

```diff
@@ -201,51 +201,54 @@ func _ready():
 		Item.new(36, "Mono Blade", Item.ItemTier.LEVEL_THREE, Item.ItemType.MOVE, "You swing the mono balde.", Moves.MoveList.MELEE_T3),
 		Item.new(37, "Psychic Immolation", Item.ItemTier.LEVEL_THREE, Item.ItemType.MOVE, "Let them live in the flames!", Moves.MoveList.PSY_T3),
 		Item.new(38, "Healing Nanites", Item.ItemTier.LEVEL_THREE, Item.ItemType.MOVE, "Healing yourself is below you.", Moves.MoveList.HEAL_T3),
+		
+		# Unique Items
+		Item.new(39, "Boss Key", Item.ItemTier.LEVEL_ZERO, Item.ItemType.KEY, "This key probably opens something!", null),
```

Comment:
Could we make these giant object arrays into a Json file we auto load?

Reply:
Not in time for the submission.

I created a ticket for it: #224